market/plugin-data-profiler/1.0.0/spec.json

```diff
@@ -2,11 +2,11 @@
   "specVersion": "1.0",
   "label": "Data Profiler",
   "description": "Generate statistics for each column or field within the data feed. Supports logical, informational, quantitative, uniques statistics generation",
-  "author": "Cask",
-  "org": "Cask Data, Inc.",
+  "author": "CDAP",
+  "org": "CDAP",
   "created": 1510006834,
```

Comment:
should this be updated? Or will that be part of the marketplace P?

Reply:
It will be part of market

Reply:
It will be part of market